### DISCUSSION TITLE
Doodba's Python versioning policy, given upstream Odoo's own testing doesn't seem to use their official supported version

### DISCUSSION THREAD

**Discussion by theangryangel** [Tecnativa/doodba#428] [Doodba - The image] [answered]
In https://github.com/odoo/odoo/pull/73074 an Odoo employee openly states that 12.0 doesn't use 3.5 for running tests. I'm sure there are other examples of this, but I've not gone looking yet.

Given that, is the Doodba policy of sticking to the official supported version relevant? Is it more pragmatic to just use whatever version they actually use to test Odoo?

**Comment by pedrobaeza** [comment, marked as answer]
That's not exactly the same. They use Python 3.6 for runbot/testing, but the official Python version aligned with the existing Debian version when 12.0 came out is 3.5, and requirements.txt is adjusted according this. It's true that if they don't test such version, errors may arise. Anyways, this should be last version using such Python. Given the age, I wouldn't touch the base version.
